src/components/BigTopBanner/BigTopBanner.tsx

```diff
@@ -0,0 +1,32 @@
+import {memo} from 'react';
+
+import {Container, Button} from '@material-ui/core';
+
+import useStyles from './BigTopBanner.styles';
+
+import {CommontWalletIcon} from 'components/atoms/Icons';
```

Comment:
Please change from `Commont` to `Common`

Reply:
resolve in https://github.com/myriadsocial/myriad-web/pull/1515/commits/c1d083d95c731d2f4b2b5a0fff1f7acc652debed